src/test/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueCloseTest.java

```diff
@@ -42,18 +44,18 @@ public void testTailAfterClose() {
     @Test
     public void reacquireAppenderAfterClose() {
         try (final ChronicleQueue queue = SingleChronicleQueueBuilder.builder(getTmpDir(), WireType.BINARY).build()) {
-            final ExcerptAppender appender = queue.acquireAppender();
+            final ExcerptAppender appender = acquireThreadLocalAppender(queue);
```

Comment:
Use createAppender instead (in try/finally above)

Reply:
This test requires the thread local unfortunately

Reply:
Ah yes